### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by cs-utkarsh-surana**
#### UTCs:  

- [x] Connector installation verified.  
- [x] Connector logo verified.  
- [x] Actions and Playbook list verified.  
- [x] Playbook tags verified.  
- [x] All Playbook are in info mode verified.  
- [x] All Playbook are in inactive mode verified.

### CODE REVIEW

File: trend-micro-cloud-app-security/info.json [hunk 1]
```diff
@@ -0,0 +1,878 @@
...
+          "title": "Report ID",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "description": "Specify the ID of the Virtual Analyzer report contained in the va_report_link field returned by the Get Security Log API.",
+          "name": "report_id"
+        }
+      ],
+      "output_schema": {},
```
Comment (cs-omkar-kumbhar) [inline trend-micro-cloud-app-security/info.json:186]: output_schema is missing

File: trend-micro-cloud-app-security/info.json [hunk 2]
```diff
@@ -0,0 +1,878 @@
...
+        {
+          "title": "Account Provider",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "select",
+          "description": "Provider of the protected service.",
+          "name": "account_provider",
+          "options": [
+            "office365",
```
Comment (cs-omkar-kumbhar) [inline trend-micro-cloud-app-security/info.json]: first alphabet should be in uppercase

File: trend-micro-cloud-app-security/info.json [hunk 3]
```diff
@@ -0,0 +1,878 @@
...
+            "delete"
+          ]
+        },
+        {
+          "title": "Senders",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "description": "A csv of sending email addresses",
```
Comment (cs-omkar-kumbhar) [inline trend-micro-cloud-app-security/info.json]: update this description

File: trend-micro-cloud-app-security/info.json (hunk 1 above)
Comment (cs-utkarsh-surana) [reply trend-micro-cloud-app-security/info.json:186]: This is uncertified

**Review by cs-omkar-kumbhar — APPROVED** [review]

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**